containermetrics/cpu_spike_handler.go

```diff
@@ -0,0 +1,73 @@
+package containermetrics
```

Comment:
I think the better names for the files would be cpu_spike_reporter.go, stats_reporter.go and reporters_runner.go which takes in a list of reporters. `handler` is usually used in API requests handler and this confused me for a second. Would like to hear from the rest of @cloudfoundry/cf-diego